src/man/sssd.conf.5.xml

```diff
@@ -3406,6 +3406,45 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>dns_resolver_server_timeout (integer)</term>
+                    <listitem>
+                        <para>
+                            Defines the amount of time (in milliseconds)
+                            SSSD would try to talk to DNS server before
+                            trying next DNS server.
```

Comment:
Hi, I read this a couple times but it's not really clear to me what it means. 

I interpret this as `SSSD would try to talk to a DNS server for 1 second before trying a next DNS server`, but why would SSSD only communicate with a DNS server for 1 second? Does it mean SSSD connects to a DNS server but only waits for a response for 1 second?

Reply:
I think that the timer is only used to resolve the DNS server, once it's resolved the timer no longer applies.

Reply:
I have rephrased the complete sentence. Can you pls check the latest changes.

Also I have split the dns_resolver_* parameters from “failover.xml” and created a new file “dns_options.xml” to sync the parameter definition since currently the definition for the parameter “dns_resolver_timeout” is different in files “failover.xml” and “sssd.conf.5.xml”.

It will help maintaining the dns_resolver_* parameters definition from a single file.